crates/control/src/motion/look_at.rs

```diff
@@ -150,28 +139,23 @@ impl LookAt {
 
 fn look_at(
     joint_angles: Joints<f32>,
-    ground_to_zero_head: Isometry3<Ground, Head>,
     camera_matrices: &CameraMatrices,
-    offset_in_image: Point2<Pixel>,
+    pixel_target: PixelTarget,
```

Comment:
Why did you change this?

Reply:
We wanted to move away from having to specify specific pixel values for the camera size we currently have and abstract that in a separate type. Therefore the `ImageRegionTarget::default()` (new name for PixelTarget) doesn't have to be changed everywhere in the calls of `LookAt` and can now be managed via the type, while the values are now given in one location in the `default.json`.

Reply:
Okay, makes sence